client/lib/support/support-user/test/localstorage-bypass.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import {
+	key,
+	clear,
+	setItem,
+	getItem,
+	removeItem,
+	length,
+} from '../localstorage-bypass';
+
+describe( 'localstorage-bypass', () => {
```

Comment:
To enable this test in the single test runner, we'll need to whitelist this by adding:
```json
		"support": {
			"support-user": {
				"test": [ "localstorage-bypass" ]
			}
		},
```

to `client/tests.json`.

Also see: https://github.com/Automattic/wp-calypso/blob/master/test/README.md